src/cr/cube/cube.py

```diff
@@ -363,6 +378,14 @@ def unweighted_counts(self):
         """
         return self._measures.unweighted_counts.raw_cube_array[self._valid_idxs]
 
+    @lazyproperty
+    def valid_count(self):
+        """ndarray of valid counts, valid elements only."""
+        # --- In case of ndim > 2 the sum should be done on the second axes to get the
+        # --- correct sequence of valid count (e.g. CA_SUBVAR).
+        axes = 1 if len(self._all_dimensions) > 2 else 0
+        return np.sum(self._measures.means.valid_count[self._valid_idxs], axis=axes)
```

Comment:
We shouldn't sum here. Because we lose information that way. Either add a "summary" property, for the valid counts, or do the summation from the exporter code. I'd prefer the summary method.

We might still sometimes need the verbose valid count, and to suggest that they're always collapsed is not correct.